app/src/main/assets/html/error/file-not-found.html

```diff
@@ -0,0 +1,24 @@
+<!doctype html>
+<html lang="en">
+    <meta charset="utf-8">
```

Comment:
should these pages have a `<meta http-equiv="Content-Security-Policy" content="default-src chrome:">` (per the pages in https://dxr.mozilla.org/mozilla-central/source/docshell/resources/content/netError.xhtml)?